hera_qm/utils.py

```diff
@@ -132,6 +135,75 @@ def get_metrics_ArgumentParser(method_name):
                        'visibilities formed with these antennas will be set to True.')
         a.add_argument('filename', metavar='filename', nargs='*', type=str, default=[],
                        help='file for which to flag RFI (only one file allowed).')
+    elif method_name == 'delay_xrfi_run':
```

Comment:
Are you familiar with argument groups? e.g. https://github.com/HERA-Team/hera_cal/blob/7ee5af6186e278ef8cd627cc81b2853e3e62d88a/hera_cal/smooth_cal.py#L430

Might make this whole thing more intelligible.

Reply:
I saw you doing that in hera cal. It's just an organizational thing right? Doesn't actually change behavior?
Are you suggesting using it within one of these "methods", or to replace the whole if/else block?

Reply:
It's purely organizational. And it makes the -h much neater.